browser-switch/src/main/java/com/braintreepayments/api/BrowserSwitchClient.java

```diff
@@ -47,36 +47,36 @@ public BrowserSwitchClient() {
      * @param activity             the activity used to start browser switch
      * @param browserSwitchOptions {@link BrowserSwitchOptions} the options used to configure the browser switch
      */
-    public void start(@NonNull FragmentActivity activity, @NonNull BrowserSwitchOptions browserSwitchOptions) throws BrowserSwitchException {
-        assertCanPerformBrowserSwitch(activity, browserSwitchOptions);
-
-        Context appContext = activity.getApplicationContext();
-
-        Uri browserSwitchUrl = browserSwitchOptions.getUrl();
-        int requestCode = browserSwitchOptions.getRequestCode();
-        String returnUrlScheme = browserSwitchOptions.getReturnUrlScheme();
-
-        JSONObject metadata = browserSwitchOptions.getMetadata();
-        BrowserSwitchRequest request =
-                new BrowserSwitchRequest(requestCode, browserSwitchUrl, metadata, returnUrlScheme, true);
-        persistentStore.putActiveRequest(request, appContext);
-
-        if (activity.isFinishing()) {
-            String activityFinishingMessage =
-                    "Unable to start browser switch while host Activity is finishing.";
-            throw new BrowserSwitchException(activityFinishingMessage);
-        } else if (browserSwitchInspector.deviceHasChromeCustomTabs(appContext)) {
-            boolean launchAsNewTask = browserSwitchOptions.isLaunchAsNewTask();
-            customTabsInternalClient.launchUrl(activity, browserSwitchUrl, launchAsNewTask);
-        } else {
-            Intent launchUrlInBrowser = new Intent(Intent.ACTION_VIEW, browserSwitchUrl);
-            try {
-                activity.startActivity(launchUrlInBrowser);
-            } catch (ActivityNotFoundException e) {
-                throw new BrowserSwitchException("Unable to start browser switch without a web browser.");
-            }
-        }
-    }
+//    public void start(@NonNull FragmentActivity activity, @NonNull BrowserSwitchOptions browserSwitchOptions) throws BrowserSwitchException {
```

Comment:
This will be removed in a future PR, just commented it out here because it was causing conflicts with the other `start` method in the demo app

Reply:
I'd vote for going ahead and removing this since we aren't gonna need it going forward. Is there a strong reason to keep?

Reply:
No there's just a separate ticket to remove all the unused methods - I can remove this one here just to keep it clean